### ISSUE TITLE
Theme preferences  [Bug?]

### ISSUE THREAD

**Issue by alizteralvord** [owl4ce/dotfiles#175]
**Describe the bug**
Theme sync not automatically switch between light and dark.

**Expected behavior**
When use Eyecandy / Mechanical  automatically switch between day and night themes

**Screenshots**
![image](https://user-images.githubusercontent.com/67093484/163332322-1e4899e1-9e80-4b1d-9da1-5d390da55ee4.png)

**Desktop (please complete the following information):**
 - **OS**: Arch Linux

**Additional context**
this is clean install, what i did only changed the settings for `tint2` (only change panel items position) and removed the `openbox` margins, the other settings are still the same.

**Comment by owl4ce** [comment]
**Hi @alizteralvord**

I think this is a common issue, especially when we are using custom environment (not DE).

I don't understand how to get the behavior to work as expected, does it work in an environment with dark mode features like Windows, Android, etc(?) What's clear, my (dotfiles) environment doesn't have that function to synchronize the theme in the website with the OS .. because I don't understand how it works.

**Comment by owl4ce** [comment]
Actually I have no problem with that. 😄

**Comment by owl4ce** [comment]
Hmm ..
https://forum.manjaro.org/t/auto-dark-theme-on-websites-according-to-system-setting/43721/18

**Comment by owl4ce** [comment]
Also ..
https://bugs.chromium.org/p/chromium/issues/detail?id=998903

**Comment by alizteralvord** [comment]
> Actually I have no problem with that. smile

Me too, I always use dark mode all the time, only when creating web pages I need this feature, but that's okay

**Comment by alizteralvord** [comment]
i don't know why, but now it work 
![a](https://user-images.githubusercontent.com/67093484/163680310-a885acab-5f80-417d-8c30-8899a70d0516.gif)

**Comment by owl4ce** [comment]
Yeah, I see **firefox** works like a charm.  That's why I referenced the **chromium** issue earlier.
